OutOfSchool/OutOfSchool.WebApi/Controllers/RatingController.cs

```diff
@@ -0,0 +1,194 @@
+
+        /// <summary>
+        /// Add a new rating to the database.
+        /// </summary>
+        /// <param name="dto">Rating entity to add.</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        [Authorize(Roles = "parent,admin")]
+        [ProducesResponseType(StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPost]
+        public async Task<IActionResult> Create(RatingDto dto)
+        {
+            var rating = await service.Create(dto).ConfigureAwait(false);
+
+            if (rating == null)
+            {
+                return BadRequest("Can't create a Rating with such parameters!\n" +
+                    "Please check that entity, parent, type information are valid and don't exist in the system yet.");
+            }
+
+            return CreatedAtAction(
+                nameof(GetById),
+                new { id = rating.Id, },
+                rating);
+        }
+
+        /// <summary>
+        /// Update info about a specific rating in the database.
+        /// </summary>
+        /// <param name="dto">Rating to update.</param>
+        /// <returns>Rating.</returns>
+        [Authorize(Roles = "parent,admin")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpPut]
+        public async Task<IActionResult> Update(RatingDto dto)
+        {
+            var rating = await service.Update(dto).ConfigureAwait(false);
+
+            if (rating == null)
+            {
+                return BadRequest("Can't change Rating with such parameters\n" +
+                    "Please check that id, entity, parent, type information are valid and exist in the system.");
+            }
+
+            return Ok(rating);
+        }
+
+        /// <summary>
+        /// Delete a specific Rating entity from the database.
+        /// </summary>
+        /// <param name="id">Rating's id.</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        [Authorize(Roles = "admin")]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status500InternalServerError)]
+        [HttpDelete("{id}")]
+        public async Task<IActionResult> Delete(long id)
+        {
+            CheckIncomingId(id);
+
+            await service.Delete(id).ConfigureAwait(false);
+
+            return NoContent();
+        }
+
+        private void CheckIncomingId(long id)
```

Comment:
please name it ValidateId like in other controllers

Reply:
Dear @dmitryy90, this was done.